browser/net/brave_network_delegate_base.h

```diff
@@ -20,6 +27,18 @@ namespace net {
 class URLRequest;
 }
 
+namespace brave {
+  const std::unordered_set<std::string> kTrackableSecurityHeaders = {
```

Comment:
Objects with non-trivial destructors are generally banned by CC (as well as `unordered_maps`), its better (but a bit wordy) to use `static const base::NoDestructor<base::flat_set<base::StringPiece>>`

For example https://cs.chromium.org/chromium/src/components/google/core/common/google_util.cc?q=google_util&sq=package:chromium&g=0&l=137